### ISSUE TITLE
Add aggregate statistics about review assignments

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#6130]
**Describe the problem you would like to solve**
One of the Plan S requirements is to provide "basic statistics" that are "published annually" about the number of reviews requested and received.

It's not yet clear exactly what Plan S expects, but some statistics will need to be provided.

**Describe the solution you'd like**
The following [suggestion from the forum](https://forum.pkp.sfu.ca/t/summary-review-statistics/61846/6?u=natewr) provided an outline of overall aggregate stats:

> In 2020, 451 articles were submitted, 305 were rejected, 107 were accepted and 115 were published. In total, 678 reviews were requested and 499 were received. For the 115 articles published in 2020, the average time since submission was 201 days.

**Who is asking for this feature?**
See [forum](https://forum.pkp.sfu.ca/t/summary-review-statistics/61846/6).

**Additional information**
It may be possible to fulfil the Plan S terms by using the existing Reviews Report tool. If so, it may be enough for us to identify and document how to do it that way.

**Comment by willinsky** [comment]
PEER REVIEW STATS: Here is one version of what a journal might report on peer-review that would be compliant with Plan S requirements. 

<img width="533" alt="Screen Shot 2020-10-07 at 8 44 50 PM" src="https://user-images.githubusercontent.com/10342108/95412502-f409a700-08dd-11eb-98c1-d7f9ac526bc2.png">

We might consider having a section under About called Journal Statistics that would include a free text section along with a number of optional plugin modules that can be activated from the dashboard for Journal Statistics, including (a) Peer Review; (b) Leading Articles; etc., allowing for other custom developments to be added (e.g., Global Readership)

**Comment by NateWr** [comment]
Thanks @willinsky. Where's that example from?

**Comment by willinsky** [comment]
This is from head, Nate, as I find it helpful in thinking about what should be included and how things should be identified and laid out to mock up a version, which I inevitably want to further modify after living with it for a while.

**Comment by kshuttle** [comment]
This question is related to one of the statistics requirements of Plan S but is separate from the review assignment stats, so please let me know if this should be posted elsewhere.
I am working on some documentation for users regarding Plan S requirements and noted "the average time between submission and publication" is one of the required stats.
I've investigating the possible steps for retrieving this statistic and found that the article report does not currently include "Date published," making it impossible (as far as I can see) to calculate the time between submission and publication. The closest alternative date would be "Last modified," but this reflects the most recent version of an article and not the original publication date.
As far as I can see (with thanks to @bozana for helping with this), the other stats required by Plan S could be drawn out either through the Editorial Activity report, or by combining stats in the article and review reports. For example, it's possible to filter for "Published" articles in the article report and combine that with the number of items in the review report's "Date notified" column (for "reviews requested") and the number in "Date Completed" column (for "reviews received"). 
If "Date published" was included in the article report it would be possible to filter the data to see only published articles, and then use a simple formula in excel to calculate Date Published minus Date Submitted to see number of days from submission to publication. The number of days could then be averaged to get the stat required by Plan S.

**Comment by NateWr** [comment]
Yes, can you file a new feature request issue for this? In the additional
comments, can you also comment on behalf of PS about whether the Plan S
requirements are important enough to clients that this feature needs to be
be back ported to 3.3 LTS or if 3.4 or later is sufficient.

**Comment by bozana** [comment]
I've just seen this issue: https://github.com/pkp/pkp-lib/issues/7691.
So, no need to create a new one?

**Comment by NateWr** [comment]
Yes, @kshuttle you may want to add information to #7691 regarding PS's priorities for 3.3 vs 3.4.
